docs/ossec_alerts.rst

```diff
@@ -530,6 +530,11 @@ securedrop@freedom.press for help.
 Uncommon OSSEC Alerts
 ~~~~~~~~~~~~~~~~~~~~~
 
+SecureDrop also runs automatic checks for rare situations like data
```

Comment:
no need to mention that it's rare if we're already in the uncommon alerts section :smiley:  - only submission integrity is being checked so this could read something like  "for submission data integrity issues."